physics/MP/TEMPO/mp_tempo.F90

```diff
@@ -0,0 +1,1007 @@
+!>\file mp_tempo.F90
```

Comment:
@AndersJensen-NOAA @dustinswales Why are we introducing code duplication with the first PR of this scheme? The same files show up in the MP/TEMPO directory of ccpp/physics and within the tempo submodule. Which files are intended to be used?

Reply:
Are the files actually the same? If not, which should be reviewed?

Reply:
@grantfirl The files in MP/TEMPO/tempo/drivers/ccpp should be removed.